src/tokens.ts

```diff
@@ -5,12 +5,14 @@ import { IAddress } from "./interface";
 export class FungibleTokenOfAccountOnNetwork {
     identifier: string = "";
     balance: BigNumber = new BigNumber(0);
+    plainTokenData: any = {};
 
     static fromHttpResponse(payload: any): FungibleTokenOfAccountOnNetwork {
         let result = new FungibleTokenOfAccountOnNetwork();
 
         result.identifier = payload.tokenIdentifier || payload.identifier || "";
         result.balance = new BigNumber(payload.balance || 0);
+        result.plainTokenData = payload;
```

Comment:
Perhaps we can use a generic name, that is common / consistent across all objects (if there will be others) where we want to provide the "raw" / "plain" data, as well? E.g. `plainData` / `rawResponse` / `rawPayload` (without `token`)?

Reply:
I think rawResponse fits the best here - I would use that if you agree